packages/components/src/dropdown-menu-v2/index.tsx

```diff
@@ -64,15 +76,19 @@ export const DropdownMenu = ( {
 			<DropdownMenuPrimitive.Trigger asChild>
 				{ trigger }
 			</DropdownMenuPrimitive.Trigger>
-			<DropdownMenuPrimitive.Portal>
+			<DropdownMenuPrimitive.Portal container={ portalContainer }>
 				<DropdownMenuStyled.Content
 					side={ side }
 					align={ align }
 					sideOffset={ sideOffset }
 					alignOffset={ alignOffset }
 					loop={ true }
 				>
-					{ children }
+					<DropdownMenuPrivateContext.Provider
+						value={ { portalContainer } }
```

Comment:
Any reason why we memoize the context value in #51097 but not here?

Reply:
Not really. I worked on #51097 after working on this PR, and realized that memoizing the context's value could have been an improvement in rendering performance.

I'm going to merge this PR as-is, and then decide on whether we should memoize the whole context (or not) after rebasing #51097